CP-CPS.md

```diff
@@ -222,12 +222,17 @@ No stipulation.
 
 ### 3.2.2 Authentication of organization identity
 
-Prior to issuance of a Subscriber Certificate, ISRG uses at least one of the following methods to validate the Applicant's control of each requested FQDN:
+Prior to issuance of a Subscriber Certificate, ISRG uses at least one of the following methods to validate the Applicant's control of each requested DNS name:
```

Comment:
Why this change from FQDN to DNS name?

Reply:
Because we also use these methods to validate the Applicant's control of Wildcard Domain Names, which are not FQDNs.